main.py

```
@@ -1,6 +1,118 @@
+    "md_to_rst": MarkdownToRstTool,
+    "md_to_pptx": MarkdownToPptxTool,
+    "md_to_codeblock": MarkdownToCodeblockTool,
+    "md_to_xlsx": MarkdownToXlsxTool,
+    "md_to_csv": MarkdownToCsvTool,
+    "md_to_json": MarkdownToJsonTool,
+    "md_to_latex": MarkdownToLatexTool,
+}
+
+
+@app.post("/convert")
+def convert(req: ConversionRequest, _: HTTPAuthorizationCredentials = Depends(authorize)):
```

Ruff (0.12.2)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)